acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb

```diff
@@ -9,8 +11,10 @@
   if not puppet_version
     fail "PUPPET_LEGACY_VERSION is not set, e.g. '3.7.5'"
   end
-  install_package host, 'puppet', puppet_version
+  hosts = [host]
+  install_puppet({:version => puppet_version})
```

Comment:
@joshcooper I filed a ticket for this last night: https://tickets.puppetlabs.com/browse/BKR-263